.github/config/evergreen-arm-softfp.json

```diff
@@ -12,23 +24,26 @@
       "platform":"evergreen-arm-softfp",
       "target_platform":"evergreen-arm-softfp",
       "target_cpu":"target_cpu=\\\"arm\\\"",
-      "extra_gn_arguments":"use_asan=false"
+      "extra_gn_arguments":"use_asan=false",
+      "bootloader_extra_gn_arguments": "sb_is_evergreen_compatible=true",
```

Comment:
I think the comma at the end of this line is giving you issues, it's not valid JSON